### PR TITLE
Update changelog items for v1.54.0

### PR DESCRIPTION
**Pull request by marians**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
...
+### Changed
 
-- Fix CRD API version for EKS infrastcture ref in cluster CR template. 
+- Enable the `login` command to create a client certificate for older workload clusters on Azure.
 
-### Changed
+### Fixed
 
-- When generating a kubeconfig, look for the generated secret in org namespace and fallback to default namespace to support legacy Azure clusters.
+- Fix the problem where the `template cluster` output for a v20 Cluster API cluster on AWS contained a bad infrastructure reference, resulting in ...
```
Comment (marians) [inline CHANGELOG.md:18]: @calvix Need your input here. What problem does this solve?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,13 +9,13 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [1.54.0] - 2021-12-03
 
-### Fixed
+### Changed
 
-- Fix CRD API version for EKS infrastcture ref in cluster CR template. 
+- Enable the `login` command to create a client certificate for older workload clusters on Azure.
```
Comment (marians) [inline CHANGELOG.md:14]: @whites11 Can we name the release versions, to make "older" more specific?

Comment (whites11) [reply CHANGELOG.md:14]: All existing releases are "older". Any new upcoming release will be "newer".
If you want, > 16.0.2 will need this version of kubectl-gs

Comment (whites11) [reply CHANGELOG.md:14]: But no such release exist yet

**Comment by marians** [comment]
Approval, anyone?

**Review by whites11 — APPROVED** [review]

**Pull request merged**